doc/source/installing_daemons.rst

```diff
@@ -4,20 +4,142 @@ Installing Rucio daemons
 Prerequisites
 ~~~~~~~~~~~~~
 
-Rucio server runs on Python 2.6, 2.7.
+The Rucio daemons run on Python 2.6 and 2.7 on any Unix-like platform.
```

Comment:
2.7 only. I mean they might run on 2.6, but we don't test against that anymore.

Reply:
Servers and clients as well?